src/AutoRest.CSharp/MgmtTest/AutoRest/MgmtTestTarget.cs

```diff
@@ -19,6 +23,12 @@ internal class MgmtTestTarget
         public static void Execute(GeneratedCodeWorkspace project, CodeModel codeModel, SourceInputModel? sourceInputModel)
         {
             Debug.Assert(codeModel.TestModel is not null);
+            Debug.Assert(Configuration.MgmtConfiguration.TestModeler is not null);
+
+            // reads all the source code files from the disk
+            if (Configuration.MgmtConfiguration.TestModeler.SourceCodePath == null)
```

Comment:
Shall we assign a default value for TestModeler.SourceCodePath as `Configuration.OutputFolder/..`?   In most case we may don't need the user to configure it.

Reply:
I am considering to remove this, because the things related with this is not used yet.
And this is causing the CI to fail @lirenhe FYI

Reply:
this is fixed now.

Reply:
let me start the validation pipeline on RPs